AppCenterDataStorage/AppCenterDataStorageTests/MSDataStoreTests.m

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT License.
 
 #import "MSAppCenter.h"
```

Comment:
I think this line can be removed since we use `MSAppCenterPrivate.h`

Reply:
They are actually not importing each others.